Project.toml

```diff
@@ -11,9 +11,9 @@ SuiteSparse = "4607b0f0-06f3-5cda-b6b1-a6196a1729e9"
 TSVD = "9449cd9e-2762-5aa3-a617-5413e99d722e"
 
 [compat]
-IterativeSolvers = ">= 0.8.0"
-OSQP = ">= 0.3.0"
-TSVD = ">= 0.3.0"
+IterativeSolvers = "0.8"
+OSQP = "0.3, 0.4, 0.5, 0.6"
+TSVD = "0.3, 0.4"
```

Comment:
These are needed since dependencies versions must be upper bounded in order for packages to be automatically registered. See some context here: https://discourse.julialang.org/t/please-be-mindful-of-version-bounds-and-semantic-versioning-when-tagging-your-packages/30708

Reply:
We should probably install the compat helper bot. It will create a PR whenever a dependency releases a new major version. (And we should then freely release patch versions of our package). It is very annoying for a user if it installs OSQP first, gets the newest version, and then is unable to install ProximalOperators.